hack/components/bump-linux-bridge.sh

```diff
@@ -49,13 +56,65 @@ COPY --from=builder ${LINUX_BRIDGE_PATH}/bin/tuning ${LINUX_BRIDGE_TAR_CONTAINER
 RUN sha256sum ${LINUX_BRIDGE_TAR_CONTAINER_DIR}/bridge >${LINUX_BRIDGE_TAR_CONTAINER_DIR}/bridge.checksum
 RUN sha256sum ${LINUX_BRIDGE_TAR_CONTAINER_DIR}/tuning >${LINUX_BRIDGE_TAR_CONTAINER_DIR}/tuning.checksum
 EOF
-    ${OCI_BIN} build -t ${LINUX_BRIDGE_IMAGE_TAGGED} .
+}
+
+check_and_create_docker_builder() {
+    existing_builder=$(docker buildx ls | grep -w "$DOCKER_BUILDER" | awk '{print $1}' || true)
+    if [ -n "$existing_builder" ]; then
+        echo "Builder '$DOCKER_BUILDER' already exists. Using existing builder."
+        docker buildx use "$DOCKER_BUILDER"
+    else
+        echo "Creating a new Docker Buildx builder: $DOCKER_BUILDER"
+        docker buildx create --driver-opt network=host --use --name "$DOCKER_BUILDER"
+    fi
+}
+
+build_docker_image() {
+    docker buildx build --platform "${PLATFORMS}" --build-arg BUILD_ARCH="$ARCH" -t "${LINUX_BRIDGE_IMAGE_TAGGED}" . --push
+    docker buildx rm "$DOCKER_BUILDER"
+}
+
+build_podman_image() {
+    podman manifest rm "${LINUX_BRIDGE_IMAGE_TAGGED}" || true
+    podman rmi "${LINUX_BRIDGE_IMAGE_TAGGED}" || true
+    podman manifest create "${LINUX_BRIDGE_IMAGE_TAGGED}"
+
+    for platform in "${PLATFORM_LIST[@]}"; do
+        podman build --no-cache --build-arg BUILD_ARCH="$ARCH" --platform "$platform" --manifest "${LINUX_BRIDGE_IMAGE_TAGGED}" .
```

Comment:
why no-cache ?
we do want cache locally (fast build is important)

Reply:
As discussed no-cache is removed to speed up build.